src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/conditionDataProviders/parentConditionDataProvider.js

```diff
@@ -13,6 +13,12 @@ export default function(data: Object, dataPath: ?string): {[string]: any} {
 
     do {
         parentDataPath = parentDataPath.substring(0, parentDataPath.lastIndexOf('/'));
+
+        if (!jsonpointer.has(data, parentDataPath)) {
+            currentConditionData.__parent = null;
```

Comment:
@alexander-schranz not sure if we should return here `null` or `data` like in line 7 when no `dataPath` is set

Reply:
Lets keep it currently with `null`.